### PR TITLE
Student discount - docs

### PR DESCRIPTION
**Pull request by Swafox**

### CODE REVIEW

File: docs/sales-billing/student.md [hunk 1]
```diff
@@ -4,4 +4,13 @@ title: Student Discount
 sidebar_label: Student Discount
 ---
 
-## TODO - Student Discount
+![Subscribing](https://i.imgur.com/JWmOSir.png)
+
+## Getting the discount
+20% student discount is guaranteed to accounts created using a student e-mail address.
+Tryhackme supports all student e-mail addresses and automatically recognizes domains like **.edu** and **.ac.uk**
```
Comment (Kent55) [inline docs/sales-billing/student.md]: Can you make this `StudlyCaps`, i.e. TryHackMe instead of Tryhackme.

File: docs/sales-billing/student.md [hunk 2]
```diff
@@ -4,4 +4,13 @@ title: Student Discount
...
+
+## Troubleshooting
+In case if your student email was not automatically recognized and the website is displaying *$10 / £7.70* as the payment amount, please send an email to support@tryhackme.com using your student e-mail and mention your username.
```
Comment (Kent55) [inline docs/sales-billing/student.md]: The indentation just needs slightly aligning here (line 14/15).

File: docs/sales-billing/student.md (hunk 1 above)
Comment (Swafox) [reply docs/sales-billing/student.md]: Fixed that.

Comment (Kent55) [reply docs/sales-billing/student.md]: LGTM :tada:

**Pull request merged**